src/Centreon/Application/Controller/Administration/ParametersController.php

```diff
@@ -84,15 +90,21 @@ public function getParameters(): View
         $refreshInterval = '';
         $isAcknowledgementPersistent = false;
         $isAcknowledgementSticky = false;
+        $isAcknowledgementNotify = false;
+        $isAcknowledgementWithServices = false;
```

Comment:
should not be true by default ?

Reply:
you are totally right, this has been fixed